### ISSUE TITLE
Move the documentation for the custom hooks from README

### ISSUE THREAD

**Issue by aromalanil** [aromalanil/useCustomHooks#25]
## Issue Description
Currently, the documentation for every hooks are added to the README file. This can get complex as the number of hooks in the package increases.

## Possible Solution
We should move these to a new location. A separate doc section would help. Please share your suggestion on how to solve this.

**Comment by GopikrishnanSasikumar** [comment]
@aromalanil one way is to create a directory `docs` in the root level and put the documentations there in a markdown file there. 

Creating a sample structure will be great. So contributors can use that structure.

**Comment by aromalanil** [comment]
@GopikrishnanSasikumar so should we create separate markdown for each hook or should we include all in one file. Also, how can we mention the documentation in the README? Will a simple redirection link do the job?

**Comment by GopikrishnanSasikumar** [comment]
> @GopikrishnanSasikumar so should we create separate markdown for each hook or should we include all in one file. Also, how can we mention the documentation in the README? Will a simple redirection link do the job?

All in one file. You can create a `readme.md` file in the docs directory so it will appear when one open `docs`. 

Then you can add a link to documentations in main readme.

**Comment by aromalanil** [comment]
I have moved the documentation to the `docs` folder.
But it is not complete. I have added complete documentation for the `useLocalStorage` hook, but other hook lacks parameters and return values in their docs.

To contribute look the `useLocalStorage` as an example, and add parameters and return values to all other hooks. 
The code for each hooks have documented the parameters and return values as per JSdoc specification, refer that for details.
